fork/module-to-cdn/modules.json

```diff
@@ -221,15 +221,6 @@
       }
     }
   },
-  "ally.js": {
-    "var": "ally",
-    "versions": {
-      ">= 1.0.0": {
-        "development": "/ally.min.js",
-        "production": "/ally.min.js"
-      }
-    }
-  },
```

Comment:
I would let it here in terms of backward compatiliby, in case you still have it in the code base it can break your app.
Remove it from the code and you should not have it anymore in the final application without this change.